nh_clinical/tests/res_user/test_check_location_ids_constraint.py

```diff
@@ -15,8 +15,6 @@ def setUp(self):
         self.user_pool = self.env['res.users']
         self.patient_pool = self.env['nh.clinical.patient']
         self.allocating_pool = self.env['nh.clinical.allocating']
-        self.resp_allocation_pool = \
-            self.env['nh.clinical.user.responsibility.allocation']
```

Comment:
As for the comment above, if this file has not been moved to another repo, and the deleted lines have not been moved to another file, what do these changes entail?

Reply:
I just noticed that it wasn't being used in the test so removed it.

Reply:
Ah cool. I take it's a common setup so it might have been fully copy/pasted between test files? If it is not being used then all good :)